tests/unit/clients/jobManagerClient.spec.ts

```diff
@@ -191,6 +194,62 @@ describe('JobManagerClient', () => {
           expect(postFun).toHaveBeenCalledTimes(1);
           expect(response).toStrictEqual(expectedResponse);
         });
+
+        it('should create Export job successfully with MIXED tiles strategy', async () => {
```

Comment:
Why not pass strategy to test instead of creating two separate tests?

Reply:
fixed